reference/docs-conceptual/learn/tutorials/01-discover-powershell.md

```diff
@@ -0,0 +1,287 @@
+
+```powershell
+Get-Process | Get-Member -MemberType Method
+```
+
+When you get a response back, PowerShell usually only displays a few properties. In the above response, `Name`, `MemberType` and `Definition` was displayed. To change that display, you can use the cmdlet `Select-Object`. `Select-Object` allows you to specify what columns you want to see. You can either provide it with the name of the column, a comma separated list or a wildcard `*`. Here's an example where `Select-Object` is used to retrieve `Name` and `Definition`:
+
+```powershell
+Get-Process | Get-Member | Select-Object Name, Definition
+```
+
+### Search by type
+
+Another way to approach searching for the command you want, is by searching for commands all operating on the same type. When you used `Get-Member`, you got the returned type as the first line of the response like so:
+
+```output
+TypeName: System.Diagnostics.Process
+```
+
+You can now use this type and search for commands like so:
+
+```powershell
+Get-Command -ParameterType Process
+```
+
+The result from invoking the above is a list with commands that solely operates on the `Process` type:
+
+```output
+CommandType     Name                                               Version    Source
+-----------     ----                                               -------    ------
+Cmdlet          Debug-Process                                      7.0.0.0    Microsoft.PowerShell.Managem…
+Cmdlet          Enter-PSHostProcess                                7.1.0.0    Microsoft.PowerShell.Core
+Cmdlet          Get-Process                                        7.0.0.0    Microsoft.PowerShell.Managem…
+Cmdlet          Get-PSHostProcessInfo                              7.1.0.0    Microsoft.PowerShell.Core
+Cmdlet          Stop-Process                                       7.0.0.0    Microsoft.PowerShell.Managem…
+Cmdlet          Wait-Process                                       7.0.0.0    Microsoft.PowerShell.Managem…
+```
+
+As you can see, knowing the type of a command, can greatly narrow down your search after commands that might be interesting for you.
+
+## Exercise - calling your first command
+
+In this exercise, you'll learn how to run your first command.
+
+1. Start a PowerShell console by typing `pwsh`:
+
+   ```powershell
+   pwsh
+   ```
+
+1. Run the following `$PSVersionTable.PSVersion`:
+
+   ```powershell
+   $PSVersionTable.PSVersion
+   ```
+
+   Your output looks similar to this:
+
+   ```output
+   Major  Minor  Patch  PreReleaseLabel BuildLabel
+   -----  -----  -----  --------------- ----------
+   7      1      0 
+   ```
+
+   Congrats, you've successfully run your first command and was able to get information of what version of PowerShell is installed on your system.
+
+## Exercise - find related commands
```

Comment:
I alreeady did Get-Process in the concept part. Thinking this should be something else. Ideas @sdwheeler ?